File: legacy/src/com/cattailsw/nanidroid/util/UIUtil.java
package com.cattailsw.nanidroid.util;

import android.os.Build;

public class UIUtil {

    public static boolean isAfterEclair(){ // 2.1
	return Build.VERSION.SDK_INT > Build.VERSION_CODES.ECLAIR_MR1;
    }

    public static boolean isGingerbread() { // 2.3
	return Build.VERSION.SDK_INT > Build.VERSION_CODES.FROYO;
    }

}

File: src/main/kotlin/com/cattailsw/nanidroid/util/UIUtil.kt
package com.cattailsw.nanidroid.util

import android.os.Build

/** SDK predicates retained as Java-callable static APIs for existing callers. */
object UIUtil {
    @JvmStatic
    fun isAfterEclair(): Boolean = Build.VERSION.SDK_INT > Build.VERSION_CODES.ECLAIR_MR1

    @JvmStatic
    fun isGingerbread(): Boolean = Build.VERSION.SDK_INT > Build.VERSION_CODES.FROYO
}
